stage_descriptions/geospatial-02-ck3.md

```diff
@@ -0,0 +1,63 @@
+In this stage, you'll add support for validating the latitude and longitude provided in the `GEOADD` command.
+
+### The `GEOADD` command (Validate coordinates)
+Latitudes and longitudes used in the `GEOADD` command should be in a certain range as per [EPSG:3857](https://epsg.io/3857). Valid longitudes are from -180 to 180 degrees. Valid latitudes are from -85.05112878 to 85.05112878 degrees. Both of these limits are inclusive.
+Example use case:
```

Comment:
> If either of these values aren't within the appropriate range, `GEOADD` returns an error. Examples: 


(We need to explain what we're showing examples of - just "example use case" doesn't fully explain what's happening)